test/framework/client/collect.go

```diff
@@ -238,11 +270,20 @@ func CollectResponse(
 		var err error
 		appPodName, err = framework.PodNameOfApp(cluster, opts.application, opts.namespace)
 		if err != nil {
-			return types.EchoResponse{}, err
+			return "", "", err
 		}
 	}
 
-	stdout, stderr, err := cluster.Exec(opts.namespace, appPodName, container, cmd...)
+	return cluster.Exec(opts.namespace, appPodName, container, cmd...)
+}
+
+func CollectResponse(
```

Comment:
Not introduced by this PR but this should actually probably be called `CollectEchoResponse` and the other just `CollectResponse`